### DISCUSSION TITLE
mc vs mcli? Which one is better?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [minio/minio#16808] [Q&A] [answered]
mc vs mcli? Which one is better? 
What is the difference?

**Comment by vadmeste** [comment, marked as answer]
@martinenkoEduard they are the same thing. Some users like to rename mc to mcli locally if they use 'Midnight Commander' (mc) tool in their machines.

**Comment by krauthosting** [comment]
On Debian based systems `mc` in PATH does conflict with the Debian package `mc`.  
Having both `/usr/bin/mc` vs `/usr/local/bin/mc` in the PATH is calling for trouble.  
Hence for Debian systems the `mc` package is shipping `/usr/local/bin/mcli`
https://dl.min.io/client/mc/release/linux-amd64/mc.deb
FYI `mcli` is close but separated to `mmcli` from the `modemmanager` package.
